### PR TITLE
csm-1.2 backport: CASMINST-4712: Update outdated cray-conman references

### PR DESCRIPTION
**Pull request by mharding-hpe**
Backport of https://github.com/Cray-HPE/docs-csm/pull/1801

### CODE REVIEW

File: operations/kubernetes/Retrieve_Cluster_Health_Information_Using_Kubernetes.md [hunk 1]
```diff
@@ -21,134 +22,81 @@ ncn-w002   Ready    <none>                 8d    v1.20.13
...
-### Retrieve a List of Healthy Pods
+### Retrieve a list of unhealthy pods
 
 ```bash
-ncn# kubectl get pods -A | grep -e 'Completed|Running'
+ncn# kubectl get pods -A | grep -Ev 'Completed|Running'
 ```
 
-Example output:
+or
```
Comment (cdelatte-hpe) [inline operations/kubernetes/Retrieve_Cluster_Health_Information_Using_Kubernetes.md:51]: I am confused by this "or"

File: operations/kubernetes/Retrieve_Cluster_Health_Information_Using_Kubernetes.md [hunk 2]
```diff
@@ -21,134 +22,81 @@ ncn-w002   Ready    <none>                 8d    v1.20.13
...
-default           kube-keepalived-vip-mgmt-plane-nmn-local-zzc4n     1/1     Running      1    2d11h
-istio-system      grafana-ff8b4b964-grg8m                            1/1     Running      0    2d11h
-istio-system      istio-citadel-699fc7bcf-rn99p                      1/1     Running      0    2d11h
-istio-system      istio-ingressgateway-59bf97fbc5-h6x4w              1/1     Running      0    2d11h
-istio-system      istio-ingressgateway-hmn-5d7777c75-2c5nz           1/1     Running      0    2d11h
-istio-system      istio-ingressgateway-hmn-5d7777c75-dh4fl           1/1     Running      0    2d11h
-istio-system      istio-init-crd-10-1.2.10-wgg99                     0/1     Completed    0    2d11h
-istio-system      istio-init-crd-11-1.2.10-dcp4h                     0/1     Completed    0    2d11h
+```bash
+ncn# kubectl get pods -A | grep -E 'Creating|ImagePull|Error|Init|Crash'
```
Comment (cdelatte-hpe) [inline operations/kubernetes/Retrieve_Cluster_Health_Information_Using_Kubernetes.md:54]: This seems out of place.  Prob goes with the "or" and might be better at line 50?

**Review by cdelatte-hpe — COMMENTED** [review]
a couple comments on flow that confused me.

File: operations/kubernetes/Retrieve_Cluster_Health_Information_Using_Kubernetes.md (hunk 1 above)
Comment (mharding-hpe) [reply operations/kubernetes/Retrieve_Cluster_Health_Information_Using_Kubernetes.md:51]: It is showing two different ways to look for unhealthy pods. I will add some language to clarify this.

**Review by dlaine-hpe — APPROVED** [review]

**Review by denniswalker — APPROVED** [review]

**Pull request merged**